README.md

```
@@ -534,120 +534,263 @@ OPENCODE_ANTIGRAVITY_PID_OFFSET_ENABLED=1                 # pid_offset_enabled
+   ```
+2. Re-authenticate:
+   ```bash
+   opencode auth login
+   ```
+
+---
+
+### 403 Permission Denied (`rising-fact-p41fc`)
+
+**Error:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)